roles/filesystem/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Create default share type
+  become: true
+  command: >
+    docker exec kolla_toolbox openstack
```

Comment:
It may be beneficial to use the `kolla_toolbox: ` module, which I think performs the same function but perhaps more reliably. You could run the openstack bit as a `command` module:

```
kolla_toolbox:
  module_name: command
  module_args: |
    openstack --os-interface ...
```